### DISCUSSION TITLE
Explicit vs Implicit formulation

### DISCUSSION THREAD

**Discussion by FlyingCat2023** [Anura3D/Anura3D_OpenSource#76] [Q&A] [answered]
Hi Dr. Francesca Ceccato,

I am reading the Anura3D manual. I have a question on the explicit formulation which is used by Anura3D. I wonder if you can share the idea why an explicit formulation is used instead of an implicit one?

Best regards,
Jianlin Huang

**Comment by FrancescaCeccato** [comment, marked as answer]
Dear Jianlin Huang,
the reason is hystorical: when we started working with Anura, more than 10 years ago, we were focusing on dynamic fast problems, thus the explicit formulation was the best option. Moreover, it is easier to implement.
Later, we also worked on implicit formulation and a very first version following the work by Lars Beuth is in Anura and you can check it in the code and further develop it. There are numbers of complications in using an implicit formulation with large deformations in MPM; despite most of them have been discussed in the literature, explicit is still simpler and appropriate for most applications.
If you have ideas or want to work on this please come back to us and we can collaborate.
